02-how-to-use.Rmd

```diff
@@ -2,179 +2,95 @@
-learning and the joy of solving problems in creative and efficient ways.
+Our message is this: learning R for your education job is doable, challenging, and rewarding all at once. We wrote this book for you because we do this work every day. We're not writing as people who have mastered all there is to know about education data science. We're writing as people who learned R and data science *after* we chose education. And like you, improving the lives of students is our daily practice. Learning to use R and data science helped us do what matters most to us. Join us in enjoying all that comes with R and data science---both the challenges of learning and the joys of solving problems in creative and efficient ways.
 
 ## Different strokes for different data scientists in education
 
-As we learned in the introduction, it's tough to define data science in education 
-because people are educated in all kinds of settings and in all kinds of age groups. Education organizations
-require different roles to make it work, which creates different kinds of data
-science uses. A teacher's approach to data analysis is different from an
-administrator's or an operations manager's. 
-
-We also know that learning data science and R is not in the typical job
-description. Most readers of this book are educators working with data and
-looking to expand their tools. You might even be an educator who *doesn't*
-work with data, but you've discovered a love for learning about the lives of
-students through data. Either way, learning data science and R is probably not
-in your job description.
-
-Like most professionals in education, you've got a full work schedule and
-challenging demands in the name of improving the student experience. Your busy
-workday doesn't include regular professional development time or self-driven
-learning. You also have a life outside of work, including family, hobbies, and
-relaxation. We struggle with this ourselves, so we've designed this book to be
-used in lots of different ways. The important part in learning this material is
-to establish a routine that allows you to engage and practice the content every
-day, even if for just a few minutes at a time. That will make the content 
-ever-present in your mind and will help you shift your mindset so you start seeing
-even more opportunities for practice.
-
-We want all readers to have a rewarding experience, and so we believe there
-should be different ways to use this book. Here are some of those ways:
+As we learned in the introduction, it's tough to define data science in education because education spans many contexts and age groups. Education organizations require different roles to make them work, which creates different kinds of uses for data science skills and tools. The approach to adopting data science will differ depending on job role, while some principles will generalize across contexts. A teacher's approach to data analysis is different from an administrator's. 
```

Comment:
I wonder if we should use a different word from define here (in part because I suggested defining educational data science broadly - see https://github.com/data-edu/data-science-in-education/pull/591)

I am not sure it is hard to define in general so much as it is true that data science looks very different in different education roles --- thoughts?